artemis-server/src/main/java/org/apache/activemq/artemis/core/server/MessageReference.java

```diff
@@ -142,4 +142,7 @@ default boolean skipDelivery() {
     */
    long getPersistentSize() throws ActiveMQException;
 
+   long getQueueSequence();
```

Comment:
just a suggestion! I would just call it getSequence here... Like the reference Queue, since a reference will belong to a queue anyway. Nit-pick probably.. do it if you like the suggestion!

Reply:
another possible name would be order.... it doesn't matter if you take order or sequence. just putting  out an alternative.

Reply:
makes sense, will fix it up. thanks